cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
**question**: is this file worth keeping?

Reply:
Yes, I think that the cypress files that I kept (I already got rid of a bunch of examples and other stuff that they create on the first run) are good placeholders with comments that have value.